apps/portals/src/portal-components/csbc-home-page/style/_ecosystem.scss

```diff
@@ -17,46 +17,46 @@
     margin-bottom: 20px;
   }
   .control-container {
-    background: white;
     display: flex;
-    border: 1px solid #dcdcdc;
-    height: 450px;
+    height: 500px;
     .button-container {
       flex: 1;
       display: flex;
       flex-direction: column;
-      flex-grow: 1;
-      margin: 15px;
+      margin-top: 15px;
       button {
         text-align: left;
         color: #515359;
         font-size: 16px;
-        padding: 10px;
-        &.isSelected,
+        padding: 20px;
         &:hover {
-          color: Portal.$primary-action-color;
           background-color: color.change(
             Portal.$primary-action-color,
-            $alpha: 0.08
+            $alpha: 0.04
```

Comment:
Design doesn't seem to call for a hover state, but removing it completely would make the component worse

Reply:
I agree